.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ installtorchgpu12: &installtorchgpu12
   run:
     name: Install torch GPU and dependencies
     command: |
-      pip3 install --progress-bar off 'torch>=1.2.0'
+      pip3 install torch>=1.2.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
What is this? I don't know this syntax/option/etc.

Reply:
I extrapolated that command from here:
![Screen Shot 2019-10-14 at 2 22 59 PM](https://user-images.githubusercontent.com/18496064/66773917-2da3ff80-ee8e-11e9-8414-1ba7d6c9656d.png)

Reply:
Okay, let's use `torch==1.2.0+cu100` and verify nightly gpu runs fine. Then we can land this, and update #2080 to be about the upgrade to 1.3.0, and roping it in with the deprecate 1.0 issue.

Reply:
So I'm not sure what happened but something blew up with pip and now most tests are failing. I was trying a few things but I reset to a previous commit and now that's failing too. Could you help out a bit ?

Reply:
A couple of updates:

- It appears that a new version of pip was released today 2019-10-14 ([link](https://pip.pypa.io/en/stable/news/)) which somehow caused issues and went on to fail all tests using pip (eg. Commit id: 5c2ac44 (the commit before the comments) was working just fine. I tried changing a few things today and it seemed to broke so I reset to that id and yet it still broke).So in circle CI's config I have specified the previous version of pip for now! 

- I found the specific URL to install torch 1.3 with CUDA and have used it in my latest commit and it seems to be working.